frontend/packages/operator-lifecycle-manager/locales/en/olm.json

```diff
@@ -268,6 +268,8 @@
   "Valid Subscriptions": "Valid Subscriptions",
   "Repository": "Repository",
   "Container image": "Container image",
+  "Cluster in STS Mode": "Cluster in STS Mode",
+  "This cluster is using AWS Security Token Service to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, you will need to provide a role ARN (with an attached policy) during installation. Please see the operator description below for more details.": "This cluster is using AWS Security Token Service to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, you will need to provide a role ARN (with an attached policy) during installation. Please see the operator description below for more details.",
```

Comment:
I am not sure if this is the case here, but in the docs we avoid directional terms like "below" https://redhat-documentation.github.io/supplementary-style-guide/#accessibility-visual-info

Reply:
@opayne1 thanks for catching that! Does it work for you if I just remove `below` altogether?

```
Please see the operator description for more details.
```

Reply:
made the change